doc/examples/config_kube_client_side_certificate.rst

```diff
@@ -3,7 +3,7 @@
    $ cat /etc/commissaire/commissaire.conf
    {
      ...
-     "register-store-handler": [
+     "storage-handlers": [
        {
          "name": "commissaire.store.kubestorehandler",
```

Comment:
I think we can actually remove this file since it's specific to the old ``kubestorehandler``